plugins/source/gcp/codegen/templates/newapi_list.go.tpl

```diff
@@ -32,7 +32,11 @@ func fetch{{.SubService | ToCamel}}(ctx context.Context, meta schema.ClientMeta,
 	req := &pb.{{.RequestStructName}}{
 		{{if .RequestStructFields}}{{.RequestStructFields}}{{end}}
 	}
-	gcpClient, err := {{.Service}}.{{.NewFunctionName}}(ctx, c.ClientOptions...)
+	{{ if .ClientOptions }}
+	clientOptions := c.ClientOptions
+	clientOptions = append([]option.ClientOption{ {{ range .ClientOptions }}{{.}},{{end}} }, clientOptions...)
```

Comment:
We prepend the options, otherwise the tests won't be able to override the endpoint URL